test-network-function/tnf_config.yml

```diff
@@ -0,0 +1,43 @@
+generic:
+  containersUnderTest:
+    - namespace: default
+      podName: test
+      containerName: test
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.8
+    - namespace: default
+      podName: partner
+      containerName: partner
+      defaultNetworkDevice: eth0
+      multusIpAddresses:
+        - 10.217.0.29
```

Comment:
`tnf/partner/partner` is not actually under test, and should not be a container in this list IMO.